### DISCUSSION TITLE
How can I turn off searching the MS store

### DISCUSSION THREAD

**Discussion by Josh65-2201** [microsoft/winget-cli#1641] [General] [answered]
When I try to install something it always prioritizing the store with nothing related to what I put in even when the `list` command find it instantly. Is there a way of turning of the store search or putting it on the bottom of the search list.

**Comment by denelon** [comment, marked as answer]
I provided a few [customization tips](https://devblogs.microsoft.com/commandline/windows-package-manager-1-1/#customization-tips) in the blog announcing the Windows Package Manager 1.1 release.

**Comment by JustMyGithub** [comment]
For future reference:
winget source remove msstore

**Reply by ShayBox** [reply]
This is required, so many fake duplicate store packages designed to trick you into installing malware.

**Reply by Masamune3210** [reply]
There is no malware on either of the repositories. Horrible, perhaps maliciously designed programs, maybe, but no actual malware.

**Reply by Tranberry** [reply]
> There is no malware on either of the repositories. Horrible, perhaps maliciously designed programs, maybe, but no actual malware.

This is like saying "this lock is tamperproof".

**Reply by Masamune3210** [reply]
Thank you for replying to this almost 2-year-old issue by basically saying, "But there's always a chance."

**Reply by Tranberry** [reply]
Snark aside, it's pretty decent to point out that you can't trust random software.
